com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLightingFiltering.compute

```diff
@@ -15,17 +16,18 @@
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/Material/Builtin/BuiltinData.hlsl"
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl"
+#include "Packages/com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs.hlsl"
 
 //--------------------------------------------------------------------------------------------------
 // Inputs & outputs
 //--------------------------------------------------------------------------------------------------
 
-RW_TEXTURE3D(float4, _VBufferFilteringInput);
+TEXTURE3D(_VBufferFilteringInput);
 RW_TEXTURE3D(float4, _VBufferFilteringOutput);
 
 
-#define SIGMA_FILTER 2.0
-#define GROUP_SIZE_1D                      8
+#define GAUSSIAN_SIGMA 1.0
```

Comment:
Do we want to make this artist-controllable?

Reply:
it is something that is was raised during the initial discussions. But in practice, we do not want to expose the radius for the moment, so exposing the sigma has not been pointed relevant by the evaluating, plus it makes the overall UX less good given that it should only appear in Reprojection and Both